rd-net/Lifetimes/Lifetimes/LifetimeDefinition.cs

```diff
@@ -160,34 +189,57 @@ public bool AllowTerminationUnderExecution
       [PublicAPI] set => ourAllowTerminationUnderExecutionSlice.InterlockedUpdate(ref myState, value);
     }
     
+    public LifetimeTerminationTimeout TerminationTimeout => ourTerminationTimeoutSlice[myState];
+    
     #endregion
 
     
     
     #region Init
-    
+
     /// <summary>
-    /// Creates toplevel lifetime definition with no parent. <see cref="Status"/> will always be <see cref="LifetimeStatus.Alive"/>.
+    /// Creates toplevel lifetime definition with no parent. <see cref="Status"/> will always be <see cref="LifetimeStatus.Alive"/>.<br/>
+    /// Created definition and all its children (if not explicitly overriden) will have specified termination timeout (see <see cref="LifetimeTerminationTimeout"/>). 
     /// </summary>
-    public LifetimeDefinition() {}    
-    
+    public LifetimeDefinition(LifetimeTerminationTimeout terminationTimeout = LifetimeTerminationTimeout.Default)
```

Comment:
Please note that this breaks the API compatibility: the code compiled against older versions of Rd will likely break in runtime, and won't work without recompiling. Could you please add a parameterless override?

Reply:
ok

Reply:
It's common to have suffix "-Kind" for enums. Like  `LifetimeTerminationTimeoutKind`

Reply:
Ok, "-Kind" sounds good here.